internal/conductorr/api/api.go

```diff
@@ -2,45 +2,45 @@
 //
 // This package provides an API for communicating with the Conductorr backend.
 //
-//     Schemes: http, https
-//     BasePath: /v1
-//     Version: 1.0.0
-//     License: GPL-3.0 https://www.gnu.org/licenses/gpl-3.0.html
+//	    Schemes: http, https
+//	    BasePath: /v1
+//	    Version: 1.0.0
+//	    License: GPL-3.0 https://www.gnu.org/licenses/gpl-3.0.html
```

Comment:
my formatter keeps formatting stuff, oops

i think this actually is weirdly formatted still though

<img width="837" height="917" alt="Image" src="https://github.com/user-attachments/assets/1f12a60d-52e3-40a5-b86f-95081bff87d5" />

 `>~` is a tab char, you can check with `:set listchars=tab:>~,nbsp:_,trail:.`

Reply:
Weird. Are you just using "go fmt"?

Separately, I have no memory of what part of these docustrings are accurate

Reply:
Also the mixed indent is from even before formatting with gopls. I don't think gopls will fully fix mixed indent in comments, only in code

Reply:
Oh interesting, I would have thought that would use go fmt